### PR TITLE
[docs] docs for voyager v1.6

### PR DESCRIPTION
**Pull request by aishwarya24**

### CODE REVIEW

File: docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+
+Keep monitoring the metrics displayed for export data and import data processes. After you notice that the import of events is catching up to the exported events, you are ready to perform a cutover. You can use the "Remaining events" metric displayed in the import data process to help you determine the cutover.
+
+Perform the following steps as part of the cutover process:
+
+1. Quiesce your source database, that is stop application writes.
+1. Perform a cutover after the exported events rate ("ingestion rate" in the metrics table) drops to 0 using the following command:
+
+    ```sh
+    yb-voyager cutover initiate --export-dir <EXPORT_DIR> --prepare-for-fall-back
```
Comment (makalaaneesh) [inline docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md:361]: `yb-voyager cutover initiate --export-dir <EXPORT_DIR> --prepare-for-fall-back true`

**Comment by aishwarya24** [comment]
Closing this PR and will pull up a new one when these changes are ready to go it.

File: docs/content/preview/yugabyte-voyager/known-issues/_index.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ---
-title: Known issues with YugabyteDB Voyager
-headerTitle: Known issues and workarounds
-linkTitle: Known issues
+title: Manual review guideline for YugabyteDB Voyager
+headerTitle: Manual review guideline
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/preview/yugabyte-voyager/known-issues/_index.md:3]: @makalaaneesh - Adding to @ddhodge's questions, do we want users to check the known issues/manual guide before proceeding with migration ?

[the remark this answers is not in the document]
Comment (priyanshi-yb) [reply docs/content/preview/yugabyte-voyager/known-issues/_index.md:3]: The manual review guide is a part of the analyze-schema phase where users can check this guide for their respective source for the issues we have already discovered. Most of these are also covered as part of the analyze-schema report. But some of these which are difficult for analyze schema to detect are not covered there and users have to check this guide for that.

[the remark this answers is not in the document]
Comment (priyanshi-yb) [reply docs/content/preview/yugabyte-voyager/known-issues/_index.md:3]: Slack thread for the reference - https://yugabyte.slack.com/archives/C02A6QDFQNL/p1697442064911059

[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/preview/yugabyte-voyager/known-issues/_index.md:3]: @priyanshi-yb - So in that case, we need to link to the manual review guideline at the end of the analyze-schema phase of each migration type , like after this : https://docs.yugabyte.com/preview/yugabyte-voyager/migrate/live-migrate/#manually-edit-the-schema.

[the remark this answers is not in the document]
Comment (priyanshi-yb) [reply docs/content/preview/yugabyte-voyager/known-issues/_index.md:3]: yes, that is a nice idea, we can link.

**Pull request closed without merging**